policyengine_us/parameters/gov/states/ne/tax/income/credits/refundable.yaml

```diff
@@ -10,5 +10,13 @@ metadata:
   reference:
   - title: 2021 NE income tax form and instruction booklet
     href: https://revenue.nebraska.gov/files/doc/tax-forms/2021/f_1040n_booklet.pdf
+  - title: 316 Nebraska Administrative Code | Chapter 22 | §019.03
+    href: https://casetext.com/regulation/nebraska-administrative-code/revenue-department-of/title-316-nebraska-department-of-revenue/chapter-22-individual-income-tax-regulations/section-316-22-019-credits-against-the-nebraska-individual-income-tax
+  - title: 316 Nebraska Administrative Code | Chapter 22 | §019.21
```

Comment:
do the same with section 0.19.11 for the refundable cdcc, use nebraska law, and remove duplicate entries